package.json

```diff
@@ -115,5 +115,8 @@
       "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/test/__mocks__/fileMock.js",
       "\\.(css|less)$": "<rootDir>/test/__mocks__/styleMock.js"
     }
+  },
+  "dependencies": {
+    "babel-polyfill": "6.26.0"
```

Comment:
Also, should this be in dev dependencies with everything else?

Reply:
babel-polyfill instructions explicity say to add to dependencies (https://babeljs.io/docs/usage/polyfill#installation). However, with the object.assign polyfill I tried adding it as a dev-dependency and it seems ok.